single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -899,6 +902,47 @@ private void drawFeederInfo(String prefixId, String feederNodeId, List<Point> po
         root.appendChild(g);
     }
 
+    protected void insertBusInfo(String prefixId,
+                                 Point point,
+                                 Element root,
+                                 BusNode busNode,
+                                 GraphMetadata metadata,
+                                 DiagramLabelProvider initProvider,
+                                 DiagramStyleProvider styleProvider) {
+        Optional<BusInfo> busInfo = initProvider.getBusInfo(busNode);
+        busInfo.ifPresent(info -> {
+            drawBusInfo(prefixId, busNode, point, root, busInfo.get(), styleProvider, metadata);
+            addInfoComponentMetadata(metadata, busInfo.get().getComponentType(), false);
+        });
+    }
+
+    private void drawBusInfo(String prefixId,
+                             BusNode busNode,
+                             Point point,
+                             Element root,
+                             BusInfo busInfo,
+                             DiagramStyleProvider styleProvider,
+                             GraphMetadata metadata) {
+        Element g = root.getOwnerDocument().createElement(GROUP);
+        // Position
+        ComponentSize componentSize = componentLibrary.getSize(busInfo.getComponentType());
+        double shY = componentSize.getHeight() / 2;
+        Point origin = new Point(point.getX(), point.getY() - shY);
```

Comment:
here you choose to put the component exactly on the bus bar extremity, but I think we should add a parameter if a user wants to shift it a bit on the bus bar (like shiftX? because I think there's no use to shift it on the Y axis)